CHANGELOG.md

```diff
@@ -7,6 +7,13 @@
 - Added support for the Steam authentication flow.
 - Added a reference to Blank Starter Project in the README.
 
+### Changed
+
+- Upgraded the Worker SDK snapshot to `13.4.0-b7199-bbfe5-WORKER-SNAPSHOT`. This entails the following changes:
```

Comment:
As the actual snapshot version is an implementation detail, I suggest saying "Upgraded to a newer version of the Worker SDK" or something.

Reply:
It's also one more thing to keep updated

Reply:
Yup agreed - will adjust